### PR TITLE
DRAFT GEODE-8444: When requester's RVV equals provider's rvvGC, should do f…

### PR DESCRIPTION
**Pull request by gesterzhou**
…ullGII

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionVector.java [hunk 1]
```diff
@@ -994,13 +994,18 @@ public boolean isNewerThanOrCanFillExceptionsFor(RegionVersionVector<T> other) {
...
     if (gcVersion == null || gcVersion.longValue() == 0) {
       return true;
     } else {
       RegionVersionHolder<T> holder = new RegionVersionHolder<T>(gcVersion.longValue());
-      return !holder.isNewerThanOrCanFillExceptionsFor(otherHolder);
+      if (otherHolder == null) {
+        return false;
+      } else {
+        return otherHolder.dominates(holder)
+            && (gcVersion == requesterGCVersion || otherHolder.getVersion() > gcVersion);
```
Comment (mhansonp) [inline geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionVector.java:1007]: It seems like you are comparing gcversion to rvv in the  final comparison on this line 

> otherHolder.getVersion() > gcVersion

Comment (gesterzhou) [reply geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionVector.java:1007]: that's correct

**Comment by onichols-pivotal** [comment]
this PR appears to be abandoned, can it be closed?

**Comment by kohlmu-pivotal** [comment]
@gesterzhou is this ticket still active? Or is this PR abandoned?

**Comment by kohlmu-pivotal** [comment]
@dschneider-pivotal would you possibly be able to review this.

**Comment by onichols-pivotal** [comment]
This PR has been inactive for some while, can it be closed?

**Comment by onichols-pivotal** [comment]
This PR has been open for some while with no recent activity.  Please keep Geode tidy by closing PRs you're not actively using.

**Comment by onichols-pivotal** [comment]
closing as abandoned.  feel free to re-open or create a new PR from this branch if this picks up again.

**Pull request closed without merging**